AddOns/BasicFinancials/app/src/page/ItemChargeAssPurch.PageExt.al

```diff
@@ -0,0 +1,10 @@
+pageextension 20623 "Item Charge Ass. (Purch) BF" extends "Item Charge Assignment (Purch)"
```

Comment:
Ass. - can we come up with some other abbreviation ?

Reply:
Can you come up with some better suggestions